tests/dummy/app/templates/docs/between.md

```diff
@@ -0,0 +1,107 @@
+.tg .tg-eh2d{background-color:#ffffff;border-color:inherit;vertical-align:top}
+.tg .tg-47u2{font-weight:bold;background-color:#ffffff;border-color:inherit;vertical-align:top;text-align:left}
+.tg .tg-7g6k{font-weight:bold;background-color:#ffffff;border-color:inherit;text-align:center;vertical-align:top}
+</style>
+<table class="tg">
+  <tr>
+    <th class="tg-47u2">Name</th>
+    <th class="tg-7g6k">Initial State</th>
+    <th class="tg-47u2">Final State</th>
+  </tr>
+    <tr>
+    <td class="tg-eh2d">Inserted</td>
+    <td class="tg-eh2d">No</td>
+    <td class="tg-eh2d">Yes</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Kept</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">Yes</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Removed</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">No</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Received</td>
+    <td class="tg-eh2d">Remote</td>
+    <td class="tg-eh2d">Local</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Sent</td>
+    <td class="tg-eh2d">Local</td>
+    <td class="tg-eh2d">Remote</td>
+  </tr>
+</table>
+
+
+
+
+### Interruption Cases
+In this demonstration, the "Delete with Undo" option shows what happens when an animation is interrupted. An email that would have been a `removedSprite` becomes a `keptSprite`. 
+
+### Beacons
+You may have noticed that this demonstration uses `#animated-beacon`. For more on beacons, see [animated-beacon](../docs/api/components/animated-beacon). 
+
+In this example, emails animate between Refresh (mail icon), Trash, and the Inbox. When an email gets deleted, it is a `removedSprite`. If you refresh your inbox, the new email added is an `insertedSprite`. This scenario has two beacons, the refresh button and the trash button.
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="one"}}
+      {{#full-log-table as |fullLog|}}
+        {{logged-between-components fullLog=fullLog}}      
+      {{/full-log-table}}
+    {{/demo.example}}
+
+    {{demo.snippet 'between-components-snippet.hbs' label='between-components.hbs'}}
+    {{demo.snippet 'between-components-snippet.js' label='between-components.js'}}
+    {{demo.snippet 'sprites-snippet.css'}}
+{{/docs-demo}}
+
+### Animating Across Lists
+In this example, the office is hosting a dinner party. Everyone received an email invitation with two options "going" and "not going".  The people invited can change their response as many times as they want. If Dwight said he was going then changes his mind, he will be removed from the "going" list and added to the "not going" list. This means that Dwight is a `sentSprite` on the going list and a `receivedSprite` on the "not going" list. If Dwight changes his mind again, he would be removed from the "not going" list and added to the "going" list. In this case, Dwight would be a `sentSprite` on the "not going" list and a `receivedSprite` on the "going list". 
```

Comment:
This might be better as a bulleted or numbered list that shows each step of Dwight's decision and the animation state of his sprite.